iFixit/react-commerce
# Adjust layout background color

~~deploy_block 🚨 Blocked by #290~~

fixes #189 
 
## Changelog

- Use `black` token for header and footer background color

## QA

1. Visit the Vercel PR preview (replace vercel.app with cominor.com) 
2. Navigate to `/Store/Parts`
3. Verify that header and footer background colors are the same as ifixit.com
